portfolio-html/portfoliohome.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
A note:  It's traditional to name the homepage for an HTML site `index.html` and to put it in the root folder of the project.  Try to follow convention, unless you have a reason not to.